lib/kubernetes/src/reconciliation/reconcile.ts

```diff
@@ -480,9 +516,23 @@ export function* reconcile(
       null,
       createCollection,
       deleteCollection,
-      updateCollection
+      updateCollection,
+      notOursCollection
     );
 
+    if (notOursCollection.length !== 0) {
+      const nowMillis = new Date().getTime();
+      if (nowMillis - notOursLastLogMillis >= NOT_OURS_LOG_TTL_MILLIS) {
+        // Could log on a single line, but in practice there should only be a few of these at any time
+        notOursCollection.forEach(r => {
```

Comment:
For such things it's recommended to use a monotonic time source, especially when using long-running processes such as the controller is.

That's why I have built the `mtine()` and related functions in `@opstrace/utils`. Happy to do a follow-up patch.